Combobox/themes/bootstrap/Combobox.less

```diff
@@ -13,11 +13,12 @@
 }
 
 .d-combobox-arrow-styles() {
-	font-size: .7em;
-	right: 1em;
-	top: 50%;
-	height: 1em;
-	margin-top: -0.6em;
+	top: 0;
+	right: 0;
+	width: 28px;
+	text-align: center;
+	height: 28px;
```

Comment:
I think the current `1em` height setting is so the arrow height matches the text height (in the left part of the Combobox).   Why hardcode it to 28px?